financial-connections/src/main/java/com/stripe/android/financialconnections/FinancialConnectionsSheetViewModel.kt

```diff
@@ -48,17 +51,23 @@ internal class FinancialConnectionsSheetViewModel @Inject constructor(
     initialState: FinancialConnectionsSheetState
 ) : MavericksViewModel<FinancialConnectionsSheetState>(initialState) {
 
+    private val mutex = Mutex()
+
     init {
         if (initialState.initialArgs.isValid()) {
             eventReporter.onPresented(initialState.initialArgs.configuration)
             // avoid re-fetching manifest if already exists (this will happen on process recreations)
             if (initialState.manifest == null) fetchManifest()
+            viewModelScope.launch {
+                stateFlow.collect { logger.debug("STATE: ${it.webAuthFlowStatus}, recreated: ${it.activityRecreated}") }
```

Comment:
should we keep these logs @carlosmuvi-stripe ?

Reply:
we can remove them